source/src/main/java/org/cerberus/database/DatabaseVersioningService.java

```diff
@@ -8799,6 +8799,12 @@ public boolean isDatabaseUptodate() {
         SQLS.append("INSERT INTO `invariant` (`idname`, `value`, `sort`, `description`, `VeryShortDesc`) VALUES ('USERGROUP', 'TestStepLibrary', '115', 'Can modify Step Library and flag Step as Library.', '');");
         SQLInstruction.add(SQLS.toString());
 
+        // New design Login page
+        //-- ------------------------ 1158
+        SQLS = new StringBuilder();
+        SQLS.append("UPDATE `parameter` SET value='<a href=\"mailto:support@domain.com?Subject=Cerberus%20Account\">Support</a>' where param='cerberus_support_email'");
```

Comment:
We cannot change a public parameter like that.
That will change the potencial parameter configured by the users.
What can be done is to change the first inserted value on the database so that any new fresh installation of cerberus will have that new value defined. We need to secure though that the original SQL is modified without adding or removing any new SQL.